kong/core/balancer.lua

```diff
@@ -603,9 +649,10 @@ end
 
 
 local function init()
+
   local upstreams, err = get_all_upstreams()
   if not upstreams then
-    log(ngx.STDERR, "failed loading initial list of upstreams: ", err)
+    log(CRIT, "failed loading initial list of upstreams: " .. err)
```

Comment:
Minor, but better not make use of Lua-land string concatenation in those error messages, and rely on the variadic arguments of `ngx.log` instead